### PR TITLE
WebGPURenderer: Fix SampledTexture not correctly bound

### PR DESCRIPTION
**Pull request by RenaudRohlinger**
Related issue: #28268

**Description**

When resizing a `RenderTarget`, for example using a PostProcess pipeline with a `PassNode`, the resize will trigger `this.renderTarget.setSize()` which will dispatch `dispose` destroying the associated textures in the backend and then lose the proper binding on the GPU side:
```js
_destroyTexture( texture ) {

		this.backend.destroySampler( texture );
		this.backend.destroyTexture( texture );

		this.delete( texture );

}
```
This PR fix `SampledTexture` binding update. Related comment https://github.com/mrdoob/three.js/pull/28268#issuecomment-2095075418
With this fix, the error seems to no longer appears.

*This contribution is funded by [Utsubo](https://utsubo.com)*

### CODE REVIEW

**Comment by RenaudRohlinger** [comment]
All `E2E testing` seems broken on all recent PRs 🤷‍♂️ /cc @Mugen87

**Comment by Mugen87** [comment]
Sorry, I'm not familiar with the E2E tests so I have no idea what's going wrong. I try to disable the CI workflow manually for the moment.

**Review by Mugen87 — APPROVED** [review]

File: examples/jsm/renderers/common/Bindings.js [hunk 1]
```diff
@@ -124,20 +124,26 @@ class Bindings extends DataMap {
 
 			} else if ( binding.isSampler ) {
 
-				binding.update();
+				const updated = binding.update();
```
Comment (sunag) [inline examples/jsm/renderers/common/Bindings.js:127]: I think `NodeSampler.update()` should return `undefined`?

Comment (RenaudRohlinger) [reply examples/jsm/renderers/common/Bindings.js]: Oh Indeed! I was confused with the Sampler/Sampled name but it appears that the issue was only related to SampledTexture. I did revert my changes regarding Sampler!

**Comment by RenaudRohlinger** [comment]
@sunag Does this change seems ok with you? After 2 days of tests I can't reproduce my issue anymore when resizing the window on PostProcess setup. (I would suggest to remove the getter `needsBindingsUpdate` from `NodeSampledTexture` too).

**Comment by sunag** [comment]
> @sunag Does this change seems ok with you? After 2 days of tests I can't reproduce my issue anymore when resizing the window on PostProcess setup. (I would suggest to remove the getter needsBindingsUpdate from NodeSampledTexture too).

Exactly what I would suggest. This section, if I'm not mistaken, was supposed to load a `VideoTexture` url outside the declaration, but I don't think it will be necessary anymore.

Another thing that caught my attention. I think `const texture = binding.texture` variable should be declared after `binding.update()` to obtain the updated reference.

**Comment by RenaudRohlinger** [comment]
Make sense! Indeed `needsBindingsUpdate` wasn't being used anymore. I cleaned up `SampledTexture` too.

**Pull request merged**